tests/checkpointing/test_checkpoint_callback_frequency.py

```diff
@@ -87,7 +87,7 @@ def training_step(self, batch, batch_idx):
 
 
 @mock.patch("torch.save")
-@RunIf(standalone=True, min_gpus=2)
+@RunIf(min_gpus=2, standalone=True)
```

Comment:
For some reason, PyCharm decided to touch the order while using its refactoring tool...

If this bothers reviewers, I can undo those changes.

Reply:
it is just a distraction from the real change... :(